### PR TITLE
Travis/tests: Added new Travis-CI, new test, updated tests

### LINKED ISSUE #21: Implement travis-ci with pytest
### Issue and Steps to Reproduce

Gotta add that CI ya know.

### Version
v2

#### Expected

automatic running of tests w/ pytest

### PR DESCRIPTION
**Pull request by ginglis13**
# Description

Added `.travis.yml` file for CI. If I wrote it correctly, it should run `pytest` on every push/commit/PR/what have you. Also made a few changes:

### commandChanVim.py

Added check for rendering so that no canvas is drawn during testing.

### config.py

Added check `os.path.isdir()` to make sure that the input location is not a dir. If it is, use default.

### test_config.py

Should test against possible input location types. 

### test_autocomplete.py

Tests functionality of the autocomplete feature. 

### test_init.py

This test file is going to need a lot of work. It's difficult because so much is imported to it - I'm thinking it might be easiest to deal with this last after testing all other modules in the project. My plan is to slowly work my way up to this one. Any help is much appreciated :heart:

Fixes #21 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

This is for testing. Hopefully from now on, and after more tests are written/tests get more extensive, we can reference individual tests in this section. 

# Checklist:

- [X] My code follows the style of this project
- [X] I have performed a self-review of my own code
- [X] My code is self documenting
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new major crashes/bugs

### CODE REVIEW

**Comment by ginglis13** [comment]
Apologies for the merge conflict and mess of commits.

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 *.pyc
 *.swp
-*.txt
+log.txt
```
Comment (sambattalio) [inline .gitignore]: I think we can probably just remove .txt as a whole? Since I believe log's send with a .log extension

Comment (ginglis13) [reply .gitignore]: oops yea i'm not sure I meant to include this. I hadn't realized we already had a requirements.txt, I had made one for local dev. I'll resolve

**Comment by wtheisen** [comment]
Really great work on these testing features

File: tests/test_config.py [hunk 2]
```diff
@@ -4,23 +4,13 @@
 
 import pytest
```
Comment (sambattalio) [inline tests/test_config.py]: pytest++ nice

**Pull request merged**